index.html

```diff
@@ -2853,10 +2853,13 @@ <h3 id="docconformance-attr">
           Requirements for use of ARIA attributes in place of equivalent HTML attributes
         </h3>
         <p>
-          Unless otherwise stated, authors MAY use `aria-*` attributes in place of their HTML equivalents on HTML elements where the `aria-*` semantics would be expected. For example, authors MAY use `aria-disabled=true` on a [^button^] element, rather than the `disabled` attribute. However, authors SHOULD NOT use both the native HTML attribute and the `aria-*` attribute together, and MUST NOT use the these features together when their values are in opposition to each other. As stated in
+          Unless otherwise stated, authors MAY use `aria-*` attributes in place of their HTML equivalents on HTML elements where the `aria-*` semantics would be expected. For example, authors MAY specify `aria-disabled=true` on a [^button^] element, while also implementing the necessary scripting to functionally disable the `button`, rather than the use `disabled` attribute.
+        </p>
+        <p>
+          As stated in
```

Comment:
i'd turn this around slightly, along the lines of

```
As stated in ..., when HTML elements use <em>both</html> `aria-*` attributes and their HTML equivalents, user agents MUST ignore the WAI-ARIA attributes - the native HTML attributes take precedence. For this reason, ...
```

Reply:
@patrickhlauke updated.